scripts/staking.py

```diff
@@ -84,13 +79,25 @@
             next_ts = get_next_checkpoint_ts(
                 ledger_api, args.staking_contract_address
             )
-            if next_ts > time.time() and not args.skip_livenesss_check:
+
+            liveness_period = 24*60*60 # TODO Read from contract
```

Comment:
Will the TODO be implemented in another PR?

Reply:
Fixed